includes/subscriptions/class-wc-payments-invoice-service.php

```diff
@@ -116,41 +116,24 @@ public function maybe_record_first_invoice_payment( int $order_id, string $old_s
 	 * @param string          $customer_id            The WCPay Customer ID.
 	 * @param string          $wcpay_subscription_id  The WCPay Billing subscription ID.
 	 *
-	 * @throws \Exception When there's an error preparing the invoice item data.
+	 * @throws API_Exception When there's an error creating the invoice items on server.
 	 *
-	 * @return string[]|WP_Error Invoice item ids.
+	 * @return string[] Invoice item ids.
 	 */
 	public function create_invoice_items_for_subscription( WC_Subscription $subscription, string $customer_id, string $wcpay_subscription_id = '' ) {
```

Comment:
Can we typehint the return value here as well?

Reply:
I've fixed this up in the latest changes